lib/Signaling/BackendNotifier.php

```diff
@@ -132,7 +132,7 @@ public function roomInvited(Room $room, array $users): void {
 				// find a better way to notify existing users to update the room.
 				'alluserids' => $room->getParticipantUserIds(),
 				'properties' => [
-					'name' => $room->getName(),
+					'name' => $room->getDisplayName(''),
```

Comment:
@fancycode what do you use the name for?
Because it will be "UserA, UserB" for one-to-one rooms now, because the same message is used for both participants and I can not send different messages?